CONTRIBUTING.md

```diff
@@ -158,16 +127,7 @@ Once your merge request is approved it will enter the merge queue
 [darach]: https://github.com/darach
 
 Speaking of tests, Rust has a comprehensive test suite. More information about
```

Comment:
Does that even apply? Plus it should be tremor

Reply:
yep removing it